beetsplug/fetchart.py

```diff
@@ -447,6 +502,13 @@ def __init__(self):
         if not self.config['google_key'].get() and \
                 u'google' in available_sources:
             available_sources.remove(u'google')
+        if not self.config['fanarttv_key'].get() and \
+                u'fanarttv' in available_sources:
```

Comment:
Done.

In hindsight, the warning is probably rather obtrusive though, even with the fix. What do you think? Given that this is also mentioned in the documentation, I'm not sure whether it should stay.